apierrors/errors.go

```diff
@@ -35,11 +36,12 @@ var (
 	ErrMetadataVersionNotFound = errors.New("Version not found")
 
 	NotFoundMap = map[error]bool{
-		ErrDatasetNotFound:       true,
-		ErrEditionNotFound:       true,
-		ErrVersionNotFound:       true,
-		ErrInstanceNotFound:      true,
-		ErrDimensionNodeNotFound: true,
+		ErrDatasetNotFound:         true,
+		ErrEditionNotFound:         true,
+		ErrVersionNotFound:         true,
+		ErrInstanceNotFound:        true,
+		ErrDimensionNodeNotFound:   true,
+		ErrDimensionOptionNotFound: true,
```

Comment:
shame not to alphabetise (which I believe is the right word) these, while you're sorting things below